src/porepy/models/contact_mechanics_biot_model.py

```diff
@@ -40,8 +40,8 @@ def __init__(self, params=None):
         self.scalar_parameter_key = "flow"
 
         # Scaling coefficients
-        self.scalar_scale = 1
-        self.length_scale = 1
+        self.scalar_scale = params.get('scalar_scale', 1)
+        self.length_scale = params.get('length_scale', 1)
```

Comment:
Yes, this has been my guiding principle as well. I must admit there are quite a few additional parameters in the subclasses I'm currently using. But I think this actually reinforces my view; there are so many different possible modifications we might want to do that we can't include all.

Reply:
agreed.

Reply:
The reason I made this particular suggestion is that these parameters are set in the `__init__` method. This is contrary to any other method in `ContactMechanics` and `ContactMechanicsBiot`, which by nature of not being `__init__` are more natural to overwrite when subclassing.

#### Why?
Many constants (parameter names, etc) are set in the `__init__` method, which makes it natural to subclass, but then call by cooperative subclassing (by `super().__init__`.
Because this is set to 1, when I subclass, I will have to either
* set these to parameters __again__ after I have called `super().__init__`
OR
* bypass this `__init__` method by implementing its other functionality, then manually call `ContactMechanics.__init__()`.

The latter option is of course infeasible and super confusing. But, I would argue that the first option is rather confusing as well. I can only speak for myself however saying that this caused quite a few unexpected results at some point due to these being set back to 1 after I thought I had set them to some other value.